src/Bindle/ConnectionInfo.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Linq;
+
+namespace Deislabs.Bindle;
+
+public class ConnectionInfo
+{
+    static readonly string[] serverAliases = {
```

Comment:
Why 7 options, do we need to be compatible with some existing connection string spec?  This is going to make documentation harder than if we specify a single specific key.

Reply:
Nope. Just more of a convenience thing.